client/my-sites/hosting/cache-card/index.js

```diff
@@ -156,32 +120,35 @@ export const CacheCard = ( {
 			<CardHeading id="cache" size={ 20 }>
 				{ translate( 'Cache' ) }
 			</CardHeading>
-			<CardBody>
-				<EdgeCacheDescription>
+			<div className="flex flex-col">
+				<p className="text-base/normal mb-4">
 					{ translate( 'Manage your site’s server-side caching. {{a}}Learn more{{/a}}.', {
 						components: {
 							a: <InlineSupportLink supportContext="hosting-clear-cache" showIcon={ false } />,
 						},
 					} ) }
-				</EdgeCacheDescription>
-				<ToggleContainer>
+				</p>
+				<div className="text-sm/normal">
 					{ getEdgeCacheInitialLoading ? (
 						<EdgeCacheLoadingPlaceholder />
 					) : (
 						<>
-							<ToggleLabel>{ translate( 'Global edge cache' ) }</ToggleLabel>
+							<p className="font-semibold text-sm/normal mb-[9px]">
+								{ translate( 'Global edge cache' ) }
+							</p>
 							<ToggleControl
+								className="[&_label]:text-sm/normal"
 								disabled={ clearEdgeCacheLoading || getEdgeCacheLoading || ! isEdgeCacheEligible }
 								checked={ isEdgeCacheActive }
 								onChange={ toggleEdgeCache }
 								label={ edgeCacheToggleDescription }
 							/>
-							<Hr />
+							<hr className="my-4 w-full text-[#e0e0e0]" />
```

Comment:
For such hardcoded color and size values, is it possible to abstract them away into Tailwind configuration? There are theme-defined color variables like `--color-text-subtle`. What's the best way to use them with Tailwind classes?

Also, margins and paddings in Calypso are practically always defined in `px` units. Can Tailwind be configured to fit into the existing design system, and to expand classes like `my-4` into `px` values?

One of the main selling points for Calypso is that "we could use Tailwind to standardize on margins, fonts, etc." To make such promises credible, the willingness and capability to implement them should be demostrated from the very beginning.

Reply:
>  is it possible to abstract them away into Tailwind configuration?

Yes, there is. I just put it here as a first step in moving things forward. If we decide to merge this in, I can explore styles better to see if this color exists in our design system and put the same naming convention in the `tailwind.config.js` file.
I believe we need to take a closer look at our branding colors and define them in the tailwind config. An example of how this can be done can be found [here](https://tailwindcss.com/docs/customizing-colors#naming-your-colors)

> Can Tailwind be configured to fit into the existing design system, and to expand classes like my-4 into px values?

I'm not sure if this can be done, however, the classes I used are the equivalent of pixels.
Moreover, using `em` and `rem` is more flexible and will often work better with your responsive design than px.